Source/WebCore/dom/CustomElementRegistry.h

```diff
@@ -81,6 +83,7 @@ class CustomElementRegistry : public RefCounted<CustomElementRegistry>, public C
     HashMap<const JSC::JSObject*, JSCustomElementInterface*> m_constructorMap;
     MemoryCompactRobinHoodHashMap<AtomString, Ref<DeferredPromise>> m_promiseMap;
     MemoryCompactRobinHoodHashSet<AtomString> m_disabledShadowSet;
+    Lock m_constructorMapLock;
```

Comment:
I've temporarily removed `WTF_GUARDED_BY_LOCK` in attempt to figure out a bit weird Windows failures. Will make sure to bring back all the assertions once Windows builds. Fun.